autograph.yaml

```diff
@@ -833,6 +907,8 @@ authorizations:
           - appkey2
           - appkey3
           - normankey
+          - normandy
+          - remote-settings
```

Comment:
Could you please add a dedicated user for remote-settings? If the signer is not the first in the list of authorized signers it does not pick it

Reply:
I added a "kintodev" user that has the "remote-settings" signer.